### ISSUE TITLE
Allow for keywords on comments as well

### ISSUE THREAD

**Issue by christianlupus** [z0al/dependent-issues#463]
I want to track the dependencies of e.g. feature requests by the community. Thus, I can/want to add a comment with `Depends on #123` to make dependencies.

As far as I understand the code of this action, only the major body of the issue (aka the feature request itself) is checked for the keywords. I just get the message `No dependencies found. Running clean-up` for all issues.

If this is out of scope of this action it would be useful to state that in the action description as I thought it should work but only after having a glance at the source, I am pretty (?) sure, it is not implemented.

**Comment by z0al** [comment]
Hi @christianlupus , that's out of the scope of this action. Fun fact, 4-5 years ago I had a GitHub bot that did exactly that but the approach (using comments) had many drawbacks it ended up being unusable. I am quite happy with the current implementation and have no plans to support comments.

The good news is, that you can actually edit the description of issues/PRs as an owner of a repo. For example, I can edit this issue description even though you are the author. You can do the same to adjust the dependencies of a community request

![image](https://user-images.githubusercontent.com/12673605/177112101-f86f17ca-0b9a-4687-b879-8149e9162fe6.png)

I hope that helps.
